tests/testthat/test-param-estimates.R

```diff
@@ -38,7 +38,7 @@ withr::with_options(list(bbr.bbi_exe_path = read_bbi_path()), {
     )
   })
 
-  test_that("param_estimates correctly errors on ONLYSIM models [BBR-PEST-003]", {
+  test_that("param_estimates correctly errors on ONLYSIM models [BBR-PEST-002]", {
```

Comment:
I'm to blame for this, but while you're fixing up these IDs, do you mind also dealing with the weird "errors on" ending that you mentioned in gh-494?  Using the colon approach mentioned in gh-494, that'd be something like

```
param_estimates correctly errors: Bayesian model
param_estimates correctly errors: Bayesian model with multiple estimation methods
param_estimates correctly errors: ONLYSIM models
```